### ISSUE TITLE
No info if package has a new version.

### ISSUE THREAD

**Issue by danbruegge** [williamboman/mason.nvim#300]
### Problem description

Can't see if packages need an update. Have to click through every package manually.

### Neovim version (>= 0.7)

NVIM v0.8.0-dev-855-gcbb2e634c9
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions     -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security     -fstack-clash-protection -fcf-protection -flto=auto -Wp,-U_FORTIFY_SOURCE -Wp,-D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -DNVIM_TS_HAS_SET_ALLOCATOR -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/build/neovim-git/src/build/cmake.config -I/build/neovim-git/src/neovim-git/src -I/usr/include -I/build/neovim-git/src/build/src/nvim/auto -I/build/neovim-git/src/build/include
Compiled by builduser

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info

### Operating system/version

Linux alfred 5.19.1-arch2-1 #1 SMP PREEMPT_DYNAMIC Thu, 11 Aug 2022 16:06:13 +0000 x86_64 GNU/Linux

### I've recently downloaded the latest plugin version of mason.nvim

- [X] Yes

### Actual behavior

After checking for new packages (all packages), there is no hint if I can update a package. Need to click through every package to see if there is a new version.

### Expected behavior

A small hint, if a package has a new version.

### Healthcheck output

```shell
mason: require("mason.health").check()
========================================================================
## mason.nvim report
  - OK: neovim version >= 0.7.0
  - OK: **Go**: `go version go1.19 linux/amd64`
  - WARNING: **cargo**: not available
  - WARNING: **luarocks**: not available
  - OK: **Ruby**: `ruby 3.0.4p208 (2022-04-12 revision 3fa771dded) [x86_64-linux]`
  - OK: **RubyGem**: `3.3.15`
  - WARNING: **Composer**: not available
  - WARNING: **PHP**: not available
  - OK: **npm**: `8.17.0`
  - OK: **node**: `v18.7.0`
  - OK: **python3**: `Python 3.10.5`
  - WARNING: **pip3**: not available
  - OK: **javac**: `Ok`
  - OK: **java**: `openjdk version "1.8.0_345"`
  - WARNING: **julia**: not available
  - OK: **wget**: `GNU Wget 1.21.3 built on linux-gnu.`
  - OK: **curl**: `curl 7.84.0 (x86_64-pc-linux-gnu) libcurl/7.84.0 OpenSSL/1.1.1q zlib/1.2.12 brotli/1.0.9 zstd/1.5.2 libidn2/2.3.3 libpsl/0.21.1 (+libidn2/2.3.0) libssh2/1.10.0 nghttp2/1.48.0`
  - OK: **gzip**: `gzip 1.12`
  - OK: **tar**: `tar (GNU tar) 1.34`
  - OK: **bash**: `GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)`
  - OK: **sh**: `Ok`
  - OK: GitHub API rate limit. Used: 3. Remaining: 57. Limit: 60. Reset: Tue 16 Aug 2022 08:26:48 CEST.
```

### Screenshots

![image](https://user-images.githubusercontent.com/1296438/184806936-2c669d08-0652-4366-8045-2b9ea75f9b73.png)

![image](https://user-images.githubusercontent.com/1296438/184806924-d2843c2c-f489-4035-80e1-6a97d711aca2.png)

**Comment by danbruegge** [comment]
An alternative, I could use `U` to run an update on all packages, sadly, this is not working. I click `U` and got no response.

**Comment by danbruegge** [comment]
Oh nice. Now I need to wait for new versions. Thanks. :)

**Comment by danbruegge** [comment]
![image](https://user-images.githubusercontent.com/1296438/185584923-bec7cd23-e14d-4bdd-ade4-defc2877536f.png)
Looks good now. Thanks. :)

**Comment by fbearoff** [comment]
Is there a way to customize this so available updates are shown even if a global severity level for virtual text is set?

I only want to see "ERROR" messages from LSP as virtual text so I use the config below, I will only see the update virtual text in mason if I set the min severity level to "INFO". Am I missing a way to see the mason update virtual text without chnaging my global severity level?

Thanks

```
virtual_text = {
      source = 'always',
      prefix = '■',
      -- Only show virtual text matching the given severity
      severity = {
        -- Specify a range of severities
        min = vim.diagnostic.severity.ERROR,
      },
    },
```

**Comment by williamboman** [comment]
@fbearoff Thanks for reporting, should be fixed in #738!
